src/modules/dashboards/saga.test.ts

```diff
@@ -170,3 +188,186 @@ describe('deleteDashboard()', () => {
     });
   });
 });
+
+describe('cloneDashboard', () => {
+  const mockDate = new Date(1611151694059);
+  jest.spyOn(global, 'Date').mockImplementation(() => mockDate);
```

Comment:
I believe this mock is not required - as `lastModificationDate` could be just set to 0 in test cases.

Reply:
This mock is for the `lastModificationDate : +new Date()` set to the cloned dashboard